lib/utils/initialAppConfig.ts

```diff
@@ -15,9 +15,35 @@
 */
 
 import { MLSpaceConfig } from './configTypes';
+import * as fs from 'fs';
 
 
 export function generateAppConfig (mlspaceConfig: MLSpaceConfig) {
+    //Check for properties set in config.json and default to that value if it exists
+    let clusterConfig = undefined;
+    let applicationList = [];
+    if (fs.existsSync('lib/resources/config/cluster-config.json')) {
```

Comment:
I am still a little new to the app config, so forgive my fuzziness on this. Is this `generateAppConfig` for the initial config? If so, then why would the `cluster-config.json` not exist, since it is included in this release?

Reply:
We want to move away from using cluster-config.json while still supporting customers who have cluster-config.json in the meantime. Even once we remove cluster-config.json from our release, there may still be some customers using a cluster-config.json who skipped a release and I want to make sure we're still supporting migration for them.